pkg/pillar/cmd/zedagent/handlelookupparam.go

```diff
@@ -30,8 +30,8 @@ import (
 )
 
 const (
-	infraFileName           = identityDirname + "/infra"
-	zedserverConfigFileName = tmpDirname + "/zedserverconfig"
+	infraFileName           = types.IdentityDirname + "/infra"
```

Comment:
Is this used? There is no /config/infra on running devices.

Reply:
It is being used in Zedagent... looks like provides a backdoor:

	ACLPromisc := false
	if _, err := os.Stat(infraFileName); err == nil {
		log.Debugf("Setting ACLPromisc\n")
		ACLPromisc = true
	}